frontend/dashboard/pages/OrgContentLibrary/OrgContentLibrary.test.tsx

```diff
@@ -5,224 +5,165 @@ import { screen, waitFor } from '@testing-library/react';
+    expect(uploadCodeListForOrg).toHaveBeenCalledWith(orgName, expect.any(FormData));
+    const formData: FormData = uploadCodeListForOrg.mock.calls[0][1];
+    expect(formData.get('file')).toBe(file);
   });
 
-  it('renders success toast when onUploadCodeList is called successfully', async () => {
-    const user = userEvent.setup();
-    renderOrgContentLibraryWithCodeLists({ initialEntries: [mockOrgPath] });
-    await goToLibraryPage(user, 'code_lists');
-    const uploadCodeListButton = screen.getByRole('button', { name: uploadCodeListButtonTextMock });
-    await user.click(uploadCodeListButton);
-    const successToastMessage = screen.getByText(
-      textMock('dashboard.org_library.code_list_upload_success'),
-    );
-    expect(successToastMessage).toBeInTheDocument();
+  it('renders success toast when uploadCodeListForOrg is run successfully', async () => {
+    const uploadCodeListForOrg = jest.fn();
+    const file = new File([''], 'list.json');
+    renderOrgContentLibraryWithCodeLists({ queries: { uploadCodeListForOrg } });
+
+    retrieveConfig().codeList.props.onUploadCodeList(file);
+    await waitFor(expect(uploadCodeListForOrg).toHaveBeenCalled);
+
+    const successMessage = textMock('dashboard.org_library.code_list_upload_success');
+    expect(screen.getByText(successMessage)).toBeInTheDocument();
   });
 
   it('renders error toast when onUploadCodeList is rejected with unknown error code', async () => {
-    const user = userEvent.setup();
     const uploadCodeListForOrg = jest
       .fn()
       .mockImplementation(() => Promise.reject({ response: {} }));
-    renderOrgContentLibraryWithCodeLists({
-      initialEntries: [mockOrgPath],
-      queries: { uploadCodeListForOrg },
-    });
-    await goToLibraryPage(user, 'code_lists');
-    const uploadCodeListButton = screen.getByRole('button', { name: uploadCodeListButtonTextMock });
-    await user.click(uploadCodeListButton);
-    const errorToastMessage = screen.getByText(
-      textMock('dashboard.org_library.code_list_upload_generic_error'),
-    );
-    expect(errorToastMessage).toBeInTheDocument();
-  });
+    const file = new File([''], 'list.json');
+    renderOrgContentLibraryWithCodeLists({ queries: { uploadCodeListForOrg } });
 
-  it('calls onUploadCodeList and hides default error when handleUpload is triggered', async () => {
-    const user = userEvent.setup();
-    renderOrgContentLibraryWithCodeLists({ initialEntries: [mockOrgPath] });
-    await goToLibraryPage(user, 'code_lists');
-    const uploadCodeListButton = screen.getByRole('button', { name: uploadCodeListButtonTextMock });
-    await user.click(uploadCodeListButton);
-    expect(queriesMock.uploadCodeListForOrg).toHaveBeenCalledTimes(1);
-    expect(queriesMock.uploadCodeListForOrg).toHaveBeenCalledWith(org, expect.any(FormData));
-    const hideDefaultError = screen.queryByText(textMock('dashboard.org_library.default_error'));
-    expect(hideDefaultError).not.toBeInTheDocument();
```

Comment:
Not sure what was the intention of this test. Anyways the message it searches for will never appear at all, so it doesn't make sense to test its inexistence. Therefore I deleted this test.